ios/brave-ios/Sources/Onboarding/WelcomeFocus/Resources/FocusOnboardingExtensions.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+extension Locale {
+  /// Regions focus onboarding is shown UK and Japan
+  public var isOnboardingRegion: Bool {
```

Comment:
Lets rename this to `isRevampedOnboardingRegion`/`isNewOnboardingRegion` or something to differentiate that its displaying the updated UI

Reply:
Change at https://github.com/brave/brave-core/pull/25068/commits/1bf33a3aa7e9f4137b767034656ad2496a75a4cb